sfge/src/test/java/com/salesforce/graph/ops/ObjectFieldUtilTest.java

```diff
@@ -331,6 +338,7 @@ public void testRegroup_withMultipleDuplicatesInSingleObject() {
                         false,
                         false,
                         false,
+                        false,
```

Comment:
@jfeingold35 I'm probably missing to catch this - is there a test in grouping logic that exercises the new `userMode` boolean to see how merge works with it?

Reply:
No, I forgot to add that. Stand by.